qml/pages/VenueMapPage.qml

```diff
@@ -72,10 +72,30 @@ Rectangle {
         onClicked: {
             var query = venueCoordinate.latitude + "," + venueCoordinate.longitude
             if (BVApp.Platform.isIos || BVApp.Platform.isMacOS) {
-                Qt.openUrlExternally("https://maps.apple.com/?q=" + query)
+                // displays a modal alert sheet with the specific options. It uses an AlertDialog on Android and a UIActionSheet on iOS.
+                nativeUtils.displayAlertSheet("", ["Apple Maps", "Google Maps"], true);
             } else {
                 Qt.openUrlExternally("geo:" + query)
             }
         }
     }
+
+    // the result of the input dialog is received with a connection to the signal textInputFinished
+    Connections {
+        target: nativeUtils
```

Comment:
Unfortunately, this leads to an error in the debugger on SailfishOS:
> [W] unknown:85 - qrc:/qml/pages/VenueMapPage.qml:85: ReferenceError: nativeUtils is not defined
Although minor, I think we should fix it.
Perhaps setting something like the ```enabled``` property on the ```Connections``` object is enough?

Reply:
Done:
```diff
-        target: nativeUtils
+        // supress warning "ReferenceError: nativeUtils is not defined" on SailfishOS by setting target to "null"                                                                                             
+        target: !BVApp.Platform.isSailfish ? nativeUtils : null
```